ansible/roles/docker/templates/ubuntu1804.Dockerfile.j2

```diff
@@ -29,7 +30,11 @@ RUN apt-get update && apt-get dist-upgrade -y && apt-get install -y \
 # something in the list of package above when installed at the same time.
 RUN apt-get install -y openjdk-17-jre-headless
 
-RUN pip3 install tap2junit
+RUN update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.6 1 &&\
```

Comment:
Minor nit, can you add a space before the line continuation characters? I don't know if it causes any problems, but at least GitHub renders it oddly without the space.
```suggestion
RUN update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.6 1 && \
```